roles/qe_common/tasks/services_tests.yml

```diff
@@ -0,0 +1,26 @@
+---
+
+- name: Verify Service Running - "{{ item }}"
+  ansible.builtin.shell:
+    cmd: |
+      oc get service -n "{{ nspace }}" "{{ item }}"
+  register: output 
+  changed_when: false
+  ignore_errors: yes
+
+- name: "Set to failed service - {{ item }}"
+  ansible.builtin.set_fact:
+     exists: FAILED
+
+- name: "If service exists - {{ item }}"
+  ansible.builtin.set_fact:
+    exists: PASSED 
+  when: output.rc == 0
```

Comment:
I don't think these are necessary if we use the custom_logger.

The output from the job should have these test_results and test summary.